https://github.com/void-linux/void-packages/issues/45216

I'm tired of seeing these setfont errors. This will hopefully/eventually be upstreamed, but I'm not holding my breath.

---

diff --git a/modules.d/10i18n/console_init.sh b/modules.d/10i18n/console_init.sh
index 3fe3b673..0e19ae30 100755
--- a/modules.d/10i18n/console_init.sh
+++ b/modules.d/10i18n/console_init.sh
@@ -49,7 +49,7 @@ set_font() {
     setfont "${FONT-${DEFAULT_FONT}}" \
         -C "${1}" \
         ${FONT_MAP:+-m "${FONT_MAP}"} \
-        ${FONT_UNIMAP:+-u "${FONT_UNIMAP}"}
+        ${FONT_UNIMAP:+-u "${FONT_UNIMAP}"} 2>/dev/null
 }
 
 dev_close() {
